jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -589,11 +590,13 @@ YUI.add('user-profile', function() {
                     && state.charmList.length === 0
                     && state.envList.length === 0;
       if (isEmpty) {
+        var staticURL = this.props.staticURL || '';
+        /* eslint-disable max-len */
```

Comment:
Rather than disabling max-len, why not setup a var that's assigned to the full image src?

~~~~
var src = `${staticURL}/juju-ui/assets/images/non-sprites/empty_profile.png`;
...
src={src}
...
~~~~